packages/x-components/src/x-modules/facets-next/components/filters/editable-number-range-filter.vue

```diff
@@ -0,0 +1,463 @@
+<template>
+  <div
+    class="x-editable-number-range-filter"
+    :class="cssClasses"
+    data-test="editable-number-range-filter"
+  >
+    <!--
+        @slot Empty slot used to customize the whole component.
+          @binding {min} number - Component min value.
+          @binding {max} number - Component max value.
+          @binding {setMin} function - Component min setter.
+          @binding {setMax} function - Component max setter.
+          @binding {emitUserModifiedFilter} function - It emits the
+          `UserModifiedEditableNumberRangeFilter` X event.
+          @binding {clearValues} function - It sets component min and max values to null.
+          @binding {hasError} boolean - Returns true when there is an error with component values.
+    -->
+    <slot v-bind="{ min, max, setMin, setMax, emitUserModifiedFilter, clearValues, hasError }">
+      <!-- eslint-disable max-len -->
+      <input
+        @change="setMin($event.target.valueAsNumber)"
+        name="min"
+        type="number"
+        class="
+          x-input
+          x-editable-number-range-filter__input x-editable-number-range-filter__input--min
+        "
+        :value="min"
+        data-test="range-min"
+      />
+
+      <input
+        @change="setMax($event.target.valueAsNumber)"
+        name="max"
+        type="number"
+        class="
+          x-input
+          x-editable-number-range-filter__input x-editable-number-range-filter__input--max
+        "
+        :value="max"
+        data-test="range-max"
+      />
+      <!-- eslint-enable max-len -->
+
+      <button
+        v-if="!isInstant"
+        @click="emitUserModifiedFilter"
+        class="x-button x-editable-number-range-filter__apply"
+        :disabled="hasError"
+        data-test="range-apply"
+      >
+        <!--
+            @slot Slot used to customize the apply button content.
+        -->
+        <slot name="apply-content">✅</slot>
```

Comment:
please change the emoji by an UNICODE symbol: https://unicode-table.com/es/2713/